app/src/main/java/com/example/wearit/ui/WardrobeScreen.kt

```diff
@@ -1,12 +1,88 @@
 package com.example.wearit.ui
 
-import androidx.compose.material.Text
+import android.util.Log
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Favorite
+import androidx.compose.material.icons.filled.PlayArrow
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.wearit.R
 import com.example.wearit.model.Category
+import com.example.wearit.ui.theme.Purple500
+
 
 @Composable
 fun WardrobeScreen(
-    currentCategory: Category
+    currentCategory: Category,
+    goToHeadGears: () -> Unit,
+    goToCoats: () -> Unit,
+    goToBlouses: () -> Unit,
+    goToTshirts: () -> Unit,
+    goToTrousers: () -> Unit,
+    goToShorts: () -> Unit,
+    goToBoots: () -> Unit,
+    goToPickerScreen: () -> Unit,
+
 ) {
-    Text(text = "Current category is " + currentCategory.name)
+    val fonts = FontFamily(
+        Font(R.font.reemkufi_bold, FontWeight.Bold)
+    )
+
+    Column {
```

Comment:
Tutaj bym proponował LazyColumn zamiast Column i tam wykorzystać funkcję items, to zajmie to dużo mniej linii